src/main.rs

```diff
@@ -313,14 +304,28 @@ async fn inner_main() -> Result<(), WrappedErr> {
 	let tui_rx = tui_rx.into_stream();
 	let from_converter = from_converter.into_stream();
 
+	if let Some(page) = restored_page {
+		tui.set_page(page);
+		to_renderer
+			.send(RenderNotif::JumpToPage(page))
+			.map_err(|e| {
+				WrappedErr(format!("Couldn't tell renderer to jump to restored page: {e}").into())
+			})?;
+		to_converter
+			.send(ConverterMsg::GoToPage(page))
+			.map_err(|e| {
+				WrappedErr(format!("Couldn't tell converter to jump to restored page: {e}").into())
+			})?;
```

Comment:
I wonder if passing these 'initial page' parameters in to the renderer and converter threads when starting them up would be faster? It might save them from starting to render an irrelevant page first, but the extra time that the threads would have to wait on decoding the file from disk might negate that?

I doubt it would make much difference at all, but if you can figure out the benchmark system tdf has (don't worry, it's completely undocumented lmao, sorry abt that), you could try out both and see if either of them improve the `render_first_page` family of benchmarks? If not, though, don't worry, I can check it out later.